src/decor/java/com/enderio/decoration/client/render/PaintedBlockColor.java

```diff
@@ -39,8 +40,8 @@ public int getColor(BlockState state, @Nullable BlockAndTintGetter level, @Nulla
 
     @Override
     public int getColor(ItemStack itemStack, int tintIndex) {
-        if (itemStack.getTag() != null && itemStack.getTag().contains("BlockEntityTag")) {
-            CompoundTag blockEntityTag = itemStack.getTag().getCompound("BlockEntityTag");
+        if (itemStack.getTag() != null && itemStack.getTag().contains(NbtTags.BLOCK_ENTITY_NBT_KEY)) {
```

Comment:
Might be worth investigating if there's a vanilla constant for this, but I do like the idea of keeping all our tags together

Reply:
There is only a private one in BlockItem